tasks.py

```diff
@@ -574,7 +575,8 @@ def __create_or_update(ctx, op):
                 getenv("LOCAL_TIME_ZONE"),
                 sg_id,
                 subnet_id,
-                getenv("LAMBDA_RELEASE_ALIAS")
+                getenv("LAMBDA_RELEASE_ALIAS"),
+                getenv("LOG_NOTIFICATIONS_FILTER_LOG_LEVEL", required=False) or ""
```

Comment:
I **think** this little `getenv(...) or ""` trick will avoid the situation where an env var ends up with the string `"None"`.